src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -157,6 +157,9 @@ angular
  * @param {string=} ng-pattern Adds the pattern validator to the ngModel of the search text.
  *     See the [ngPattern Directive](https://docs.angularjs.org/api/ng/directive/ngPattern)
  *     for more details.
+ * @param {string=} md-mode Specify the repeat-mode for suggestion lists. Acceptable values include
+ *     `virtual`(md-virtual-repeat, the default behavior) and `standard` (ng-repeat). See the
+ *     `Specifying Repeat Mode` example below for mode details.
```

Comment:
The API docs actually show up at the bottom, so the example will be above. Let's just remove `below` here.